lib/worker.js

```diff
@@ -103,7 +123,24 @@ async function getLinter(filePath) {
   loadDefaultTSLint();
 
   if (config.useLocalTslint) {
-    return getLocalLinter(basedir);
+    return resolveAndCacheLinter(basedir);
+  }
+
+  if (config.useGlobalTslint) {
```

Comment:
Is `basedir` really used when using global tslint?

Reply:
It would also be an option to separately cache the global linter after the first time we fetch it, to avoid having to resolve it again for each new directory, but ultimately it would need to be inserted into the `tslintCache` Map for each directory that we've confirmed doesn't have a local install, just like on line 109 where the same is done for the packaged "fallback" install.